stl/inc/xlocale

```diff
@@ -1375,7 +1373,6 @@ private:
     _Codecvt_mode _Mode; // default: _Consume_header
 };
 
-#if defined(__cpp_char8_t) && !defined(_M_CEE_PURE)
 template <class _From, class _To>
 struct _Codecvt_guard {

```

Comment:
Your PR description said:

> Dragged up helper `_Codecvt_guard` to be used in previous versions as it requires neither C++20 nor charX_t to run.

However, I see no new uses of `_Codecvt_guard` - is this change incomplete or unnecessary?

Reply:
Guard is used in do_in in ushort and wchar implementations, in case of utf8

Reply:
Which lines? I am still confused. I see 4 uses of `_Codecvt_guard` in code below, but it's all guarded by the same `#if defined(__cpp_char8_t) && !defined(_M_CEE_PURE)` block, so I don't see why `_Codecvt_guard` needs to be extracted. Am I missing unconditional uses of `_Codecvt_guard` in another file?